tests/sympc/tensor/grad/grad_functions_test.py

```diff
@@ -459,3 +461,94 @@ def test_grad_flatten_backward(get_clients) -> None:
     res_mpc_grad = GradFlatten.backward(ctx, grad_mpc)
 
     assert np.allclose(res_mpc_grad.reconstruct(), x, rtol=1e-3)
+
+
+@pytest.mark.parametrize("power", [2, 4, 5, 1.0])
+def test_grad_pow_forward(get_clients, power) -> None:
+    parties = get_clients(4)
+    x = torch.Tensor([[1, 2, 3], [4, 5, 6]])
+
+    x_mpc = x.share(parties=parties)
+
+    ctx = {}
+
+    if isinstance(power, float):
```

Comment:
MAYBE: Create a separate test to "test" the exception path. Each test should have a single logic to be tested.